### DISCUSSION TITLE
Sort json keys with specified order

### DISCUSSION THREAD

**Discussion by zliucd** [nlohmann/json#3117] [Q&A] [answered]
Hi,

Is is possible to sort json keys with specified order? e.g., in the order when key was assigned.  When I printed json, the order seemed to be **unordered**.

Thanks a lo.t

**Comment by nlohmann** [comment, marked as answer]
See https://json.nlohmann.me/features/object_order/

**Reply by zliucd** [reply]
thanks, that's exactly I need. It is a very useful feature!
